packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -199,6 +249,89 @@ function renderWithFalsyKey(props = {}) {
+            Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
+          </Text>
+        </Content>
+      </Item>
+      <Item title="Tab 5" key="val5">
+        <Content margin="size-160">
+          <Heading>Tab Body 5</Heading>
+          <Text>
+            Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+            Id cupidatat aute id ut excepteur exercitation magna pariatur. Mollit irure irure reprehenderit pariatur eiusmod proident Lorem deserunt duis cillum mollit. Do reprehenderit sit cupidatat quis laborum in do culpa nisi ipsum. Velit aliquip commodo ea ipsum incididunt culpa nostrud deserunt incididunt exercitation. In quis proident sit ad dolore tempor. Eiusmod pariatur quis commodo labore cupidatat cillum enim eiusmod voluptate laborum culpa. Laborum cupidatat incididunt velit voluptate incididunt occaecat quis do.
+            Consequat adipisicing irure Lorem commodo officia sint id. Velit sit magna aliquip eiusmod non id deserunt. Magna veniam ad consequat dolor cupidatat esse enim Lorem ullamco. Anim excepteur consectetur id in. Mollit laboris duis labore enim duis esse reprehenderit.
+          </Text>
+        </Content>
+      </Item>
     </Tabs>
   );
 }
+
+let ManyTabs = (props = {}) => {
+  let items = [
+    {name: 'Tab 1', children: 'Tab Body 1', icon: <Dashboard size="S" />},
+    {name: 'Tab 2', children: 'Tab Body 2', icon: <Calendar size="S" />},
+    {name: 'Tab 3', children: 'Tab Body 3', icon: <Bookmark size="S" />},
+    {name: 'Tab 4', children: 'Tab Body 4', icon: <Dashboard size="S" />},
+    {name: 'Tab 5', children: 'Tab Body 5', icon: <Calendar size="S" />},
+    {name: 'Tab 6', children: 'Tab Body 6', icon: <Bookmark size="S" />}
+  ];
+
+  let [tabs, setTabs] = React.useState(items);
+  let addTab = () => {
+    let newTabs = [...tabs];
+    newTabs.push({
+      name: `Tab ${tabs.length + 1}`,
+      children: `Tab Body ${tabs.length + 1}`
+    });
+
+    setTabs(newTabs);
+  };
+
+  let removeTab = () => {
+    let newTabs = [...tabs];
+    newTabs.pop();
+    setTabs(newTabs);
+  };
+
+  return (
+    <div style={{width: '80%'}}>
+      <Tabs {...props} aria-label="Tab example" items={tabs} onSelectionChange={action('onSelectionChange')}>
+        {item => (
+          <Item
+            key={item.name}
+            textValue={item.name}
+            title={
+              <>
+                {item.icon}
+                <Text>{item.name}</Text>
+              </>
+            }>
+            <Content margin="size-160">
```

Comment:
Slightly concerned about the correctness of Content being the same for all tabs in this example. Seems like it might generate weird behavior and we shouldn't commit this as an example people might find and replicate.

Reply:
Ehh, I don't think that should be too much of a problem. People will have their own designs to follow and thus adjust the margin/content appropriately. The docs will be a better place for them to look than the storybook itself